README.md

```diff
@@ -1,20 +1,15 @@
-# AI Software Templates Helm Charts
+# AI Lab Helm Charts
 
-This project aims to convert the [AI Software Templates](https://github.com/redhat-ai-dev/ai-lab-template) functionality into helm charts. Therefore, it aims to reproduce the flow of AI Software Templates without the usage of RHDH.
-
-## Available Helm Charts
-
-At the moment one template has helm chart support and this is the [chatbot application template](https://github.com/redhat-ai-dev/ai-lab-template/tree/main/templates/chatbot). The chatbot helm chart can be found [here](/charts/ai-software-templates/chatbot/).
-
-In addition the helm chart for the application, there are also two helm charts for setting up OpenShift Pipelines detailed [below](#openshift-pipelines).
+This project is a collection of helm charts, insipired by the [AI Software Templates](https://github.com/redhat-ai-dev/ai-lab-template).
 
 ## Gitops
 
 The gitops component, handled by ArgoCD for the RHDH case, is replaced by a Kubernetes Job created by the Helm chart, where this Job:
+
 - Creates the GitHub repository for the application.
 - Copies the application source code into the new repository.
-- Copies the Tekton Pipelines As Code PipelineRun/Pipeline/Task that build new images for the application as pull requests 
-are merged and updates the Deployment of the application with the new version of the image.
+- Copies the Tekton Pipelines As Code PipelineRun/Pipeline/Task that build new images for the application as pull requests
```

Comment:
Actually we just copy the PipelineRun.  All Pipelines and Task are remote ref's to the rhdh-pipelines repo

Reply:
Addressed in [e7dfb81](https://github.com/redhat-ai-dev/ai-lab-helm-charts/pull/32/commits/e7dfb814a567efc6592b44b65aa368868f8a7adb)